dashboard/package.json

```diff
@@ -22,7 +22,7 @@
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.21",
     "lodash-es": "^4.17.21",
-    "normalize.css": "^8.0.1",
+    "modern-normalize": "^1.1.0",
```

Comment:
Do we need this direct dependency because of the css inclusion? Otherwise it looks like this is already pulled in as a dependency from react.

Reply:
Not required, but I think it worths explicitly declare it since we are using it for resetting the style. I mean, what if react drops this dep as we did with normalize.css